src/main/antora/modules/ROOT/nav.adoc

```diff
@@ -1 +1,218 @@
+
 * xref:index.adoc[]
+
+
+
+.Core Profile
+
+* Jakarta EE Contexts and Dependency Injection
+
+** xref:cdi:cdi-basic/cdi-basic.adoc[]
+
+** xref:cdi:cdi-basicexamples/cdi-basicexamples.adoc[]
+
+** xref:supporttechs:interceptors/interceptors.adoc[]
+
+* Jakarta REST
+
+** xref:websvcs:jaxrs/jaxrs.adoc[]
+
+** xref:websvcs:jaxrs-client/jaxrs-client.adoc[]
+
+** xref:websvcs:jaxrs-advanced/jaxrs-advanced.adoc[]
+
+* Jakarta JSON
+
+** xref:web:jsonb/jsonb.adoc[]
+
+** xref:web:jsonp/jsonp.adoc[]
+
+
+
+
+.Web Profile
+
+* Jakarta Validation
+
+** xref:beanvalidation:bean-validation/bean-validation.adoc[]
+
+** xref:beanvalidation:bean-validation-advanced/bean-validation-advanced.adoc[]
+
+* Jakarta Security
+
+** xref:security:security.adoc[]
+
+* Jakarta Servlets
+
+** xref:web:webapp/webapp.adoc[]
+
+** xref:web:servlets/servlets.adoc[]
+
+* Jakarta Faces
+
+** xref:web:faces-intro/faces-intro.adoc[]
+
+** xref:web:faces-facelets/faces-facelets.adoc[]
+
+** xref:web:faces-page/faces-page.adoc[]
+
+** xref:web:faces-page-core/faces-page-core.adoc[]
+
+** xref:web:faces-develop/faces-develop.adoc[]
+
+** xref:web:faces-ajax/faces-ajax.adoc[]
+
+* Jakarta WebSocket
+
+** xref:web:websocket/websocket.adoc[]
+
+* Jakarta Persistence
+
+** xref:persist:persistence-intro/persistence-intro.adoc[]
+
+** xref:persist:persistence-basicexamples/persistence-basicexamples.adoc[]
+
+** xref:persist:persistence-querylanguage/persistence-querylanguage.adoc[]
+
+** xref:persist:persistence-criteria/persistence-criteria.adoc[]
+
+** xref:persist:persistence-string-queries/persistence-string-queries.adoc[]
+
+** xref:persist:persistence-locking/persistence-locking.adoc[]
+
+** xref:persist:persistence-entitygraphs/persistence-entitygraphs.adoc[]
+
+** xref:persist:persistence-cache/persistence-cache.adoc[]
+
+** xref:supporttechs:concurrency-utilities/concurrency-utilities.adoc[]
+
+** xref:supporttechs:transactions/transactions.adoc[]
+
+* Jakarta Enterprise Beans (Lite)
+
+** xref:entbeans:ejb-intro/ejb-intro.adoc[]
+
+** xref:entbeans:ejb-gettingstarted/ejb-gettingstarted.adoc[]
+
+** xref:entbeans:ejb-basicexamples/ejb-basicexamples.adoc[]
+
+
+
+
+.Full Profile
+
+* Jakarta Mail
+
+* Jakarta Messaging
+
+** xref:messaging:jms-concepts/jms-concepts.adoc[]
+
+** xref:messaging:jms-examples/jms-examples.adoc[]
+
+* Jakarta Batch
+
+** xref:supporttechs:batch-processing/batch-processing.adoc[]
+
+
+
+
+.Advanced
+
+* Core Profile
+
+** Jakarta JSON
+
+*** xref:web:jsonp/jsonp.adoc[]
+
+*** xref:web:jsonb/jsonb.adoc[]
+
+* Web Profile
+
+** Jakarta CDI (Full)
+
+*** xref:cdi:cdi-adv/cdi-adv.adoc[]
+
+*** xref:cdi:cdi-bootstrap-se8/cdi-bootstrap-se8.adoc[]
+
+*** xref:cdi:cdi-adv-examples/cdi-adv-examples.adoc[]
+
+** Jakarta Authentication
+
+*** xref:security:security-authentication/security-authentication.adoc[]
+
+** Jakarta Authorization
+
+*** xref:security:security-authorization/security-authorization.adoc[]
+
+** Jakarta Servlets
+
+** Jakarta Faces
```

Comment:
On second thought, I think it's much clearer if this will be put back into the root Web Profile section.